### ISSUE TITLE
dcterms:creator: better defintion

### ISSUE THREAD

**Issue by DavidFichtmueller** [DiSSCo/openDS#155]
### Term Name

dcterms:creator

### Digital Object Name

DigitalSpecimen: Citation

### Feedback

Both from the definition and the context, it is unclear who would be the creator of a citation. Would this be the author of the reference publication? But if so, why would there only be one creator, as the property would only reference one and is not repeatable. Or is this supposed to be the person who added the citation to the digital object? This on the other hand doesn't seem really useful to us. As the term already uses a custom definition, please extend it, to include the intended usage for this term.

**Comment by samleeflang** [comment]
Agreed, I am switching this to an `ods:hasAgents` property. This way it can contain multiple agents.
